.evergreen/config.yml

```
@@ -21,8 +21,23 @@ functions:
       params:
         script: |-
           set -o errexit
-          chmod u+x libmongocrypt/.evergreen/*.sh
-          ./libmongocrypt/.evergreen/print-env-info.sh
+          cd libmongocrypt
+          chmod u+x .evergreen/*.sh
+          ./.evergreen/print-env-info.sh
+
+          if [ "${is_patch}" != "true" ]; then
+            # determine if we have a release tag present on HEAD
+            head_tag=$(git tag -l --points-at HEAD '[0-9].*' | tail -1)
```

```suggestion
            head_tag=$(git tag --points-at HEAD '[0-9].*' | tail -1)
```
Per [Git documentation](https://git-scm.com/docs/git-tag), `--points-at <object>` implies `--list`, so `-l` is redundant.

---

However, on older versions of Git, it is required.  For example, [here is a failure](https://evergreen.mongodb.com/task/libmongocrypt_feature_ubuntu1604_arm64_build_and_test_and_upload_a4593971dca2134d1715fcdaaa58158680c093ab_22_06_24_00_49_31) on Ubuntu 16.04 without `-l`:

```
[2022/06/24 00:53:17.598] fatal: --points-at option is only allowed with -l.
```